### DISCUSSION TITLE
getting an error with hosting webapp on pythonanywhere

### DISCUSSION THREAD

**Discussion by Qvme** [AnswerDotAI/fasthtml#496] [Q&A]
hello there i am trying to host webapp created with fasthtml on pythonanywhere, but i am getting problem with running wsgi application:
below is error log:
`2024-10-06 18:14:21,162: Error running WSGI application
2024-10-06 18:14:21,163: TypeError: Starlette.__call__() missing 1 required positional argument: 'send'`

**Comment by pydanny** [comment]
Due to its backbone being Starlette, FastHTML is an ASGI-only framework. Can you use Python Anywhere to host ASGI apps?

**Reply by Qvme** [reply]
There is an option to do so, but it's mostly from the command line only; they haven't provided a GUI for that yet. I tried doing so with the help of this article, but I have not had any success yet:

https://help.pythonanywhere.com/pages/ASGICommandLine/
